doc_source/private-clusters.md

```diff
@@ -0,0 +1,80 @@
++ Any container image that is not in Amazon ECR or a registry inside the VPC will need to be copied to Amazon ECR or a registry inside the VPC in order to be pulled\. See [Creating local copies of container images](#creating-local-copies-of-container-images)\.
++ **Endpoint private access** is required in order for the worker nodes to register with the cluster endpoint\. **Endpoint public access** is optional\.
++ You must bypass the EKS cluster introspection by providing the cluster certificate authority and cluster API endpoint to the worker nodes\.
++ You must include the VPC endpoints found at [VPC endpoints for private clusters](#vpc-endpoints-for-private-clusters)\.
++ Managed node groups is not supported with private clusters\.
++ eksctl is not supported with private clusters\.
++ AWS X-Ray is not supported with private clusters\.
++ CloudWatch Logs is supported with private clusters\. You must include the CloudWatch Logs VPC endpoint\.
++ For deploying worker nodes, [Self-managed nodes](launch-workers#self-managed-nodes) and [Amazon EKS managed node groups](launch-workers#amazon-eks-managed-node-groups) are supported\.
+  + The instances for worker nodes must have access to the VPC endpoints\. If you are creating a managed node group, the VPC endpoint security group must allow the CIDR for the subnets or you must add the created worker node security group to the VPC endpoint security group\.
++ You need to include the following to the bootstrap arguments when launching worker nodes\. This will bypass the EKS introspection and does not require access to the EKS API from within the VPC\. Substitute the *{CLUSTER_ENDPOINT}* and *{CLUSTER_CERTIFICATE_AUTHORITY}* for the values from your EKS cluster\.
+
+   ```
+   --apiserver-endpoint {CLUSTER_ENDPOINT} --b64-cluster-ca {CLUSTER_CERTIFICATE_AUTHORITY}
+   ```
+
++ [IAM roles for service accounts](iam-roles-for-service-accounts) is supported\. You must include the STS VPC endpoint\.
++ The [Amazon EBS CSI driver](ebs-csi) is supported\. When deploying, the [kustomization.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/deploy/kubernetes/overlays/stable/kustomization.yaml) file must be changed to set the container images to use the same region as the EKS cluster\.
++ The [Amazon EFS CSI driver](efs-csi) is supported\. When deploying, the [kustomization.yaml](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/deploy/kubernetes/overlays/stable/kustomization.yaml) file must be changed to set the container images to use the same region as the EKS cluster\.
++ The [Amazon FSx for Lustre CSI driver](fsx-csi) is not supported\.
++ The `aws-auth` ConfigMap must be created from within the VPC\.
++ The [ALB Ingress Controller on Amazon EKS](alb-ingress) does not work in private clusters\.
++ [AWS Fargate](fargate) is supported with private clusters. You must include the STS VPC endpoint\.
```

Comment:
Like Amelia was pointing out, STS/S3/ECR VPC endpoints are all required for EKS Fargate to function properly in an "air-gapped" environment